draft-ietf-lamps-kyber-certificates.md

```diff
@@ -483,62 +483,62 @@ key in the previous section.
+       :     C4 76 16 48 AD 76 5C F5 D8 00 6B 51 5E 90 5A 7F
+       :     0A C0 76 B0 C6 2E FA 32 81 53 E7 CA 57 01 69 9F
+       :     13 05 F1 E6 BC 6F 90 B0 E4 9B 69 35 12 B6 CE 99
+       :     2A 8B 80 16 DD FC 1A 66 2C 7E 3F 96 19 CB D8 69
+       :     DD 77 1A F3 08 96 CC D5 91 8A C6 CB 77 46 6C 5E
+       :     77 99 96 D6 7F F9 AA BC 97 50 3F 2C 7B 7E 2D 00
+       :     0D 86 45 0F B1 80 7C A4 CA BD A4 65 82 5A 31 C7
+       :     89 A1 B7 A4 91 AB 38 72 76 5D 32 0D 0B 71 92 0F
+       :     A2 13 C9 40 93 41 6B 83 B8 12 4E 69 F6 5E 62 CB
+       :     50 00 DC C3 7A A9 A0 FF F7 39 70 C4 77 2F 35 7D
+       :     24 18 9C A6 F5 30 55 68 C0 E2 37 6A 37 62 A6 8C
+       :     60 5E 56 3C 5D 20 95 72 E0 FC 75 32 CA 29 47 29
+       :     53 55 67 B5 FC 41 3C 5E 87 92 D2 46 45 36 CC 80
+       :     8F 98 AD D7 46 64 F1 41 56 6F 90 16 A9 0A 54 18
+       :     29 A9 8A 04 64 CE 41 A8 BB 44 C2 D4 FA 3C 2C 20
+       :     94 60 72 8E F1 4A 1A 7C 4C 9B 98 D1 22 03 B4 CC
+       :     35 29 16 0A 9A B2 D7 83 8F 7F F6 B5 3A E0 5A A3
+       :     1A 7D 64 6B 7A FA 6C 45 93 25 26 A3 C3 75 56 19
+       :     BE 99 4C 21 1C 2A 31 C0 5B 34 47 83 6C B2 15 0B
+       :     E1 82 9D AE 6B 04 C5 53 5C FF 54 6E 39 2B A7 97
+       :     41 17 20 F9 24 F4 90 A5 AC 54 95 F2 13 56 D5 50
+       :     B7 82 A6 4C 16 88 B6 B6 55 BC C7 84 21 97 A4 34
+       :     C2 F6 56 3B 5B 7F 09 A7 8B CC 48 82 32 78 35 61
+       :     D1 6F 4C BA B6 75 54 00 05 07 81 57 0C 66 60 4B
+       :     81 7A D1 25 22 94 73 6E 8B 01 86 1A 4B 5A 74 51
+       :     9B 8B 6F E5 14 89 A5 07 23 92 E5 87 62 6C 71 37
+       :     76 57 5D 33 80 6A 1C 8E 27 32 AF 97 C2 68 0F 51
+       :     66 63 31 C4 EB 8B BC 04 31 C4 F9 68 32 DA F1 B3
+       :     C4 55 28 FB A1 53 F6 C7 8B 1C 19 87 02 94 7C CD
+       :     33 77 27 A4 6F B5 3B A1 1D E5 CB 41 91 34 68 59
+       :     51 6C B6 AD 72 40 0F 3C F2 09 B2 36 AE F3 5A 58
+       :     0A C8 7E B3 E3 0F AF D6 69 73 CA 8A 7D D2 67 5A
+       :     F4 1F 7A 17 B6 14 33 CD 1A F8 0F 77 08 86 9F 66
+       :     54 88 49 79 80 B1 AC 10 A0 CD CB 63 6A 00 ED 86
+       :     81 B3 5E 42 91 24 CA 80 35 07 25 B8 5F 83 A5 EA
+       :     C3 A4 A3 CC 16 00 90 3E 65 29 35 60 B9 B3 36 E5
+       :     AF 0D 52 9D AC 1A 04 81 19 30 2C B7 A9 BC C1 10
+       :     B9 48 51 BF 02 11 7F 19 9D C4 85 A8 52 B7 47 3F
+       :     09 B8 31 A6 83 1D 5B 54 C0 B7 90 D2 25 CF 6B B9
+       :     2D 94 62 A2 6C DB 33 DD A5 12 3C 7A AF 0E 26 A0
+       :     B8 36 55 EE A2 8B F3 A8 07 47 25 01 8F D6 BA E4
+       :     B6 01 CF 61 BA AB 71 A7 A3 D3 51 97 A3 43 E7 4B
+       :     4A 27 2C 12 5D 54 08 96 42 6D 85 B7 95 8D 3B 38
+       :     A6 BA 98 7E C3 72 25 C7 B4 4C DB 12 DD E4 53 9B
+       :     4A B0 82 36 36 83 F0 4B F7 A0 9C C5 C4 1D FE 83
+       :     0A 1B 16 2E 0B 32 43 34 36 2F 08 4A 14 46 77 23
+       :     34 4B AD D0 00 F8 D8 C5 37 C4 8F 99 8F 05 30 7C
+       :     EB D1 ED E0 B8 1C 3B C5 9A 06 5A 1B 6D 63 B2 6C
+       :   }
 ~~~
 
```

Comment:
<aside markdown="block">
NOTE: The first byte of the BIT STRING's content, the padding byte, has been stripped from the pretty print above.
</aside>

Reply:
@danvangeest if you are happy with this note, I'll merge.